src/kbmod/search/RawImage.cpp

```diff
@@ -245,47 +245,47 @@ void RawImage::convolve_cpu(const PointSpreadFunc& psf) {
 
             float sum = 0.0;
             float psfPortion = 0.0;
-            for (int j = -psfRad; j <= psfRad; j++) {
-                for (int i = -psfRad; i <= psfRad; i++) {
+            for (int j = -psf_rad; j <= psf_rad; j++) {
+                for (int i = -psf_rad; i <= psf_rad; i++) {
                     if ((x + i >= 0) && (x + i < width) && (y + j >= 0) && (y + j < height)) {
                         float currentPixel = pixels[(y + j) * width + (x + i)];
                         if (currentPixel != NO_DATA) {
-                            float currentPSF = psf.getValue(i + psfRad, j + psfRad);
+                            float currentPSF = psf.getValue(i + psf_rad, j + psf_rad);
                             psfPortion += currentPSF;
                             sum += currentPixel * currentPSF;
                         }
                     }
                 }
             }
-            result[y * width + x] = (sum * psfTotal) / psfPortion;
+            result[y * width + x] = (sum * psf_total) / psfPortion;
         }
     }
 
     // Copy the data into the pixels vector.
     const int npixels = getNPixels();
-    for(int i = 0; i < npixels; ++i) {
+    for (int i = 0; i < npixels; ++i) {
         pixels[i] = result[i];
     }
 }
 
 void RawImage::convolve(PointSpreadFunc psf) {
-    #ifdef HAVE_CUDA
-        deviceConvolve(pixels.data(), pixels.data(), getWidth(), getHeight(), psf.kernelData(),
-                       psf.getSize(), psf.getDim(), psf.getRadius(), psf.getSum());
-    #else
-        convolve_cpu(psf);
-    #endif
+#ifdef HAVE_CUDA
+    deviceConvolve(pixels.data(), pixels.data(), getWidth(), getHeight(), psf.kernelData(), psf.getSize(),
+                   psf.getDim(), psf.getRadius(), psf.getSum());
```

Comment:
just re: earlier proposal of one-per-line-ing this we could have a inline comment or just direct named parameters that would then distinguish between `pixels.data()` and `pixels.data()` in this context.